packages/plugin/src/processor.ts

```diff
@@ -1,12 +1,32 @@
 import type { Linter } from 'eslint';
-import { parseCode } from '@graphql-tools/graphql-tag-pluck';
+import { parseCode, GraphQLTagPluckOptions } from '@graphql-tools/graphql-tag-pluck';
+import { loadConfigSync } from 'graphql-config';
 
 type Block = Linter.ProcessorFile & {
   lineOffset: number;
   offset: number;
 };
 
-const RELEVANT_KEYWORDS = ['gql', 'graphql', '/* GraphQL */'] as const;
+const defaultGraphqlConfig = loadConfigSync({ throwOnMissing: false })?.getDefault();
```

Comment:
I'd like to reuse to `loadGraphQLConfig()` method defined in `graphql-config.ts` but that requires I have the parameter `options: ParserOptions` to pass in, so manually loading the graphql-config like this is the next best option I can think of

Reply:
Nevermind I realized I could just pass in an empty `options` parameter